src/shared/ui/MovieTitle/MovieTitle.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './MovieTitle.css';
+
+export const MovieTitle = ({ imageUrl, movieName, releaseYear, genres = [], onSelect }) => {
+
+    return (
+        <div className="card">
+            <img src={process.env.PUBLIC_URL+imageUrl} alt="Avatar" />
```

Comment:
what did you put in this `.env` variable? :)

Reply:
removed)